src/Lecture1.hs

```diff
@@ -123,4 +136,13 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String
+lowerAndGreater n list = let (l, g) = go n list in fillTemplate n l g
+  where
+    fillTemplate x l g = show x ++ " is greater than " ++ show l ++ " elements and lower than " ++ show g ++ " elements"
```

Comment:
You don't need to pass `n` to `fillTemplate` as a separate argument `x`. Functions inside `where` see all the arguments of the main function.

```suggestion
    fillTemplate l g = show n ++ " is greater than " ++ show l ++ " elements and lower than " ++ show g ++ " elements"
```

The same is true for the recursive `go` function.